azure-communication-ui/demo-app/src/chat/java/com/azure/android/communication/ui/chatdemoapp/CachedTokenFetcher.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
You can place classes like Fetcher in the shared location in the main folder.
See UrlTokenFetcher for example